ui/src/components/Dom.jsx

```diff
@@ -127,6 +142,7 @@ class DomList extends React.Component {
 
     this.state = {
       hoverEl: null,
+      domView: this.props.domView,
```

Comment:
It's generally not good practice to latch props onto state.

Do we need to do this or could we just thread the props through?